docs/data-sources/cloud_backup_snapshot_export_bucket.md

```diff
@@ -35,7 +35,7 @@ In addition to all arguments above, the following attributes are exported:
 * `cloud_provider` - Name of the provider of the cloud service where Atlas can access the S3 bucket.
 * `role_id` - Unique identifier of the Azure Service Principal that Atlas can use to access the Azure Blob Storage Container.
 * `service_url` - URL that identifies the blob Endpoint of the Azure Blob Storage Account.
-* `tenant_id` - UUID that identifies the Azure Active Directory Tenant ID.
+* `tenant_id` - (Deprecated) This field is ignored; the `mongodbatlas_cloud_provider_access_authorization.azure.tenant_id` is used instead. UUID that identifies the Azure Active Directory Tenant ID.
```

Comment:
Can we add a changelog entry for this deprecation? https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md#deprecations